packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -158,7 +158,7 @@ class FuchsiaDevices extends PollingDeviceDiscovery {
   FuchsiaDevices({
     required Platform platform,
     required FuchsiaWorkflow fuchsiaWorkflow,
-    required FuchsiaSdk fuchsiaSdk,
+    FuchsiaSdk? fuchsiaSdk,
```

Comment:
Another option is for the `FuchsiaSdk` and `FuchsiaWorkflow` to more closely mirror the treatment of the `AndroidSdk` and `AndroidWorkflow` in the `FlutterDeviceManager` constructor, but I don't know whether the way that the `FlutterDeviceManager` constructor is called makes that difficult.

Reply:
Good idea.  To make that easier I need to move `fuchsiaSdk` to `globals.dart`.
https://github.com/flutter/flutter/blob/43732e45977a04e662f0d4b2612b5f95678eaa74/packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart#L19

That may cause a g3 issues depending on if that's used and how it's imported, I will make a g3fix if needed.

Reply:
This did indeed cause a FRoB failure cl/417891057